CorsixTH/Lua/config_finder.lua

```diff
@@ -414,12 +414,13 @@ unicode_font = nil -- [[X:\ThemeHospital\font.ttc]]
 savegames = nil -- [[X:\ThemeHospital\Saves]]
 
 -------------------------------------------------------------------------------
--- Levels. By default, the "Levels" directory alongside this config file will
--- be used for storing new maps / levels in. Should this not be suitable, then
--- uncomment the following line, and point it to a directory which exists and
--- is more suitable.
+-- Levels and Campaigns. By default, the "Levels" and "Campaigns" directory alongside
+-- this config file will be used for storing new maps / levels / campaigns in. Should
+-- this not be suitable, then uncomment the following lines, and point it to a directory
```

Comment:
```suggestion
-- this config file will be used for storing new maps / levels / campaigns. If this is
-- not be suitable, then uncomment the following lines, and point it to a directory.
```
I am a little lost where the maps are going, as there are two directories and three things to store.

Reply:
I don't think I paid enough attention to this textual comment. It needs to say that created maps go into the Levels folder currently.